packages/typegpu/src/core/function/fnCore.ts

```diff
@@ -176,3 +218,27 @@ export function createFnCore(
 
   return core;
 }
+
+function checkAndReturnType(
+  name: string,
+  wgslType: string | undefined,
+  jsType: unknown,
+  ctx: ResolutionCtx,
+  externals: ExternalMap,
```

Comment:
Let's remove `externals` as it's not used anymore, and put the `ctx` at the top, for consistency.
```suggestion
  ctx: ResolutionCtx,
  name: string,
  wgslType: string | undefined,
  jsType: unknown,
```